test/spec/spectest.cpp

```diff
@@ -465,55 +466,61 @@ bool SpecTest::stringContains(std::string_view Expected,
 
 void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
   spdlog::info("{} {}", Proposal, UnitName);
-  std::ifstream JSONIS(TestsuiteRoot / Proposal / UnitName /
-                       (std::string(UnitName) + ".json"s));
-  rapidjson::IStreamWrapper JSONISWrapper(JSONIS);
-  rapidjson::Document Doc;
-  auto &Allocator = Doc.GetAllocator();
-  Doc.ParseStream(JSONISWrapper);
+  auto FName =
+      (TestsuiteRoot / Proposal / UnitName / (std::string(UnitName) + ".json"s))
+          .string();
+
+  simdjson::dom::parser parser;
+  simdjson::dom::element Doc;
+  try {
+    Doc = parser.load(FName);
+  } catch (simdjson::error_code &err) {
+    ;
```

Comment:
Should it fail? If not, we should print some error messages.

Reply:
It shouldn't fail. I was using this for testing purposes. Will remove this block.